python/src/trezorlib/ethereum.py

```diff
@@ -14,10 +14,14 @@
 # You should have received a copy of the License along with this library.
 # If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.
 
+import pathlib
```

Comment:
I would again vote for `from pathlib import Path, PurePath`

Reply:
path operations are gone from ethereum.py